ecms_base/modules/custom/ecms_migration/README.md

```diff
@@ -24,6 +24,14 @@ and will use the first _h1_ tag on the page as the title of the node. It will
 continue pulling the inner html from three user defined css selectors
 and will append that content to the `field_basic_page_body` field.
 
+### eCMS Publications
+The eCMS Publication migration will generate `publication` nodes and will accept
+a Googe Sheet with the following columns:
```

Comment:
Or is this a more general publications importer for use on any/all sites as needed?
Nitpcik "Googe"

Reply:
There are two migrations happening here. The first is the Covid RSS migration, which is a separate module specifically for the Covid site. The second, which this documentation refers, is the more general import for use with any site. I started the general import before knowing of the rss feed. I kept the code in case it can be useful for another site, and added the RSS migration afterwards.

Reply:
got it